packages/common/src/ui/layout/tables/components/DataRow/DataRow.tsx

```diff
@@ -51,6 +52,7 @@ export const DataRow = <T extends DomainObject>({
               : 'inset 0 0.5px 0 0 rgba(143, 144, 166, 0.5)',
             display: 'flex',
             flex: '1 0 auto',
+            ...rowStyle,
```

Comment:
I think a relatively straight forward implementation - though maybe it could be better by specifying exactly which styles can be altered